### ISSUE TITLE
Double Gold

### ISSUE THREAD

**Issue by StuffOfSonny** [Ryan-rsm-McKenzie/QuickLootRE#68]
I've noticed a few times, particularly when lockpicking, that QuickLoot will display the contents of an inventory, but if the inventory is pulled up in full and I take the gold from it, the QuickLoot menu will still say the gold is there, and will give me the option to take it.
I'm not sure if this is purely cosmetic, or if it's actually doubling the gold, and I'm not sure if this has anything to do with any other mods I have installed.
If it helps, I started a new save (thanks, LotD) so I don't have any perks enabled that would influence the leveled lists yet.

**Comment by Ryan-rsm-McKenzie** [comment]
Are you saying that the loot menu and the vanilla menu display the same amount of gold, or that the loot menu displays more gold than the vanilla menu?

**Comment by StuffOfSonny** [comment]
The Former.
Both display the same amount, but with Gold specifically, it sometimes doesn't update after the vanilla menu has been looted-thus seemingly allowing double gold.

**Comment by StuffOfSonny** [comment]
I have come across a little bit of extra context:
It seems if I do not take EVERYTHING from a lockpickable inventory, but I do take the Gold, the Gold remains in the Quick Loot menu. This might also have something to do with how lockpicking triggers the full menu anyways, whilst briefly flashing the quick menu.

**Comment by Ryan-rsm-McKenzie** [comment]
see if the latest release helps

**Comment by StuffOfSonny** [comment]
So far so good, though I have noticed that there is a very small frameskip like event when it loads the quick menu now. 99% of the time I don't even notice it, but the 1% (usually if I have to open a Rare chest) it catches me off guard.

I call that a worthy trade off.

**Comment by Ryan-rsm-McKenzie** [comment]
Are you saying the game is hitching or the menu isn't updating quickly enough?

**Comment by StuffOfSonny** [comment]
It's very brief, (like 1/60 frames) but I think it's more of the latter. If I know to look for it I can see the encumbrance load my max first, and then the ratio, but in the case mentioned of say unlocking a rare chest it feels more like a hitch.

**Comment by Ryan-rsm-McKenzie** [comment]
I improved the performance in the latest release, see if that helps

**Comment by IanMunster** [comment]
I too saw the frame-skip in v2.3.1, haven't seen it yet in v2.3.3

**Comment by StuffOfSonny** [comment]
I haven't ran into any chests I've had previous issue with yet, but the previously mentioned "it loads my total encumbrance, then the ratio" nitpick has been dealt with.

**Comment by IanMunster** [comment]
Had a frame-skip with a chest that was rigged with a trap.

Question; should this be moved into its own issue?

**Comment by Ryan-rsm-McKenzie** [comment]
Do you have a formID for the chest?

**Comment by IanMunster** [comment]
Chest: 3F709
Traptrigger: AE4DD
Location: Valtheim Tower (first chest inside the tower)

**Comment by Ryan-rsm-McKenzie** [comment]
I think the latest update will improve the performance

**Comment by IanMunster** [comment]
Great! Will test soon and report!

**Comment by Aindar** [comment]
Is this mod supposed to work on version 1.5.97?

**Comment by CaptainKoby** [comment]
Latest release (2.4.0) caused a 100% reproduction rate of this issue for me, former releases not. Not just gold but all items in a container, even after looting the container.

**Comment by IanMunster** [comment]
Haven't seen a frame-skip for sometime!
Will open a new Issue when I see it.

**Comment by caligineux** [comment]
I've been experiencing this issue as well. It only happens on locked containers. When you unlock the container the quick loot UI will pop up with the contents until the container opening animation completes. Once the animation is done the SkyUI container view is displayed. If you pull the inventory of the container on the SkyUI screen and close out of it the quick loot UI will show again with the full original contents of the container. The only way I can get quick loot to update the container's inventory was to try to pull the nonexistsent items from it or completely unfocus the container and refocus upon it.
